libres_util/src/res_log.c

```diff
@@ -112,24 +110,31 @@ void res_log_add_message(message_level_type message_level , FILE * dup_stream ,
  * Expects fmt to be a string with potential string-formatting, and "..." is the
  * required arguments to the string-formatting.
  */
-void res_log_add_fmt_message(message_level_type message_level , FILE * dup_stream , const char * fmt , ...) {
+void res_log_add_fmt_message(message_level_type message_level,
+                             FILE * dup_stream,
+                             const char * fmt,
+                             ...) {
   if(logh==NULL)
     res_log_init_log_default(true);
 
-  if (log_include_message(logh,message_level)) {
-      char * message;
-      va_list ap;
-      va_start(ap , fmt);
-      message = util_alloc_sprintf_va( fmt , ap );
-      log_add_message( logh , message_level , dup_stream , message , true);
-      va_end(ap);
-    }
+  if (!log_include_message(logh,message_level))
+    return;
+  char * message;
+  va_list ap;
+  va_start(ap , fmt);
+  message = util_alloc_sprintf_va( fmt , ap );
+  log_add_message(logh, message_level, dup_stream, message);
+  free(message);
+  va_end(ap);
 }
 
 
-void res_log_close(){
-    if (log_is_open( logh ))
-      log_add_message( logh , false , NULL , "Exiting ert application normally - all is fine(?)" , false);
+void res_log_close() {
+    if (log_is_open(logh))
+      log_add_message(logh,
```

Comment:
I agree that many of the lines in the code are too long, and appreciate the splitting; in particular I was not aware of the possibility to split string literals like `"part1" "part2"` - that was very good to learn. However splitting the current `log_add_message( )` call to cover four vertical lines is a bit excessive in my opinion.

Reply:
> However splitting [...]  four vertical lines is a bit excessive in my opinion.

*If the arguments fit, I will acquit.*

However, I like to _either_ keep all arguments on one line, _or_ split all lines.  I am open for suggestions.

Reply:
Well - this is not important; but I would prefer the current function call to be one *one single line*

> I like to either keep all arguments on one line, or split all lines.

My take on this is to try to group related arguments  - when that makes sense - on the same line. 

Anyway - no big deal either way.

Reply:
fixed